tests/etl/test_perf_data_adapters.py

```diff
@@ -469,3 +474,13 @@ def test_framework_not_enabled(test_repository,
 
     assert 0 == PerformanceSignature.objects.all().count()
     assert 0 == PerformanceDatum.objects.all().count()
+
+
+def test_last_updated(test_repository, failure_classifications,
```

Comment:
who calls this function?

Reply:
pytest is smart enough to figure out that this is a test by the `test_` prefix, so calls it automatically. :)